tools/vagrant/README.md

```diff
@@ -131,7 +135,10 @@ Vagrant installation is self-signed. Alternatively, you can configure your
 _apihost_ to use the non-SSL interface:
 
 ```
+# Non-Windows systems:
 wsk property set --apihost http://192.168.33.16:10001 --auth `vagrant ssh -- cat openwhisk/ansible/files/auth.guest`
+# Windows systems (Powershell):
+wsk property set --apihost http://192.168.33.16:10001 --auth $(vagrant ssh -- cat openwhisk/ansible/files/auth.guest)
```

Comment:
Using $(....) should also work on Linux and Mac. 
Don’t have the distinction have the same command for all

Reply:
OK, though changed the wording to imply that Powershell should be used when the deployment is hosted in Windows (as the default cmd prompt won't correctly substitute the result of expressions).